hack/generate-manifest.sh

```diff
@@ -31,6 +31,8 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
         --np                          Generate a manifest with ClusterNetworkPolicy and Antrea NetworkPolicy features enabled
         --keep                        Debug flag which will preserve the generated kustomization.yml
         --tun (geneve|vxlan|gre|stt)  Choose encap tunnel type from geneve, gre, stt and vxlan (default is geneve)
+        --verbose-log                 Generate a manifest with increased log-level (level 4) for Antrea agent and controller.
```

Comment:
curious about why you didn't make this `--verbosity <N>`?

Reply:
Could not find a way to change log levels in manifest through kustomize other than using configmap. Or using sed, which I felt not very neat. 
In addition, '--v=0' in base yaml is good enough to make it intuitive for devs to change log-level through configmap.
https://github.com/vmware-tanzu/antrea/pull/1142#discussion_r480829719

Reply:
makes sense